### DISCUSSION TITLE
Inbound.get_shipments()

### DISCUSSION THREAD

**Discussion by TanLeSi** [saleweaver/python-amazon-sp-api#761] [Q&A] [answered]
Hello all, 

i have a problem regarding the get_shipments function from the Inbound endpoint. 
![image](https://user-images.githubusercontent.com/89908442/194845905-ab6b11f4-361f-4ff6-b3e7-bd6d58a697d6.png)

According to amazon documentation it is possible to give a list of shipment ids as argument in the function. However when i provide a list of 5 shipment ids, only the information for the last shipment id get returned, instead of information for all 5 shipment ids that i provided.
![image](https://user-images.githubusercontent.com/89908442/194846245-c5f4959e-f9bd-4207-9945-d04217002496.png)
![image](https://user-images.githubusercontent.com/89908442/194846391-89b0d321-9e0f-421c-b407-7262686f8a41.png)

Could someone please help me with this problem?
Best regards 
Thank you

**Comment by gmweinberg** [comment, marked as answer]
You might want to try sending the shipment_ids as a comma delimited string instead of a python list. I've seen that work in some other apis.

**Reply by TanLeSi** [reply]
that helps. Thanks a lot
